Shared/Connect-M365.ps1

```diff
@@ -0,0 +1,159 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Test-EXOConnection {
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    [OutputType([bool])]
+    param (
+        [Switch]$Force
+    )
+    #Validate EXO V2 is installed
+    if ((Get-Module -ListAvailable | Where-Object { $_.Name -like "ExchangeOnlineManagement" }).count -ge 1) {
```

Comment:
We are doing `Get-Module` a lot here. I would prefer to see a function that handles this.